benchmarkcommon/src/test/java/com/ss/benchmark/httpclient/HC.java

```diff
@@ -0,0 +1,23 @@
+package com.ss.benchmark.httpclient;
+
+import java.util.concurrent.CompletableFuture;
+
+public interface HC {
+
+    //All times are milliseconds unless otherwise noted
+    static final int MAX_CONNECTION_POOL_SIZE = 200;
+    static final int CONNECT_TIMEOUT = 500;
+    static final int CONNECTION_REQUEST_TIMEOUT = 1;
+    static final int READ_TIMEOUT = 2000;
+
+    void setup(String baseUrl);
```

Comment:
Let's get rid of this and change `HCPerformanceTests#getClient` to `createClient(String baseUrl)`. The impls will then just do setup stuff in their constructors.